browser/android/brave_sync_worker.cc

```diff
@@ -207,33 +189,19 @@ bool BraveSyncWorker::IsFirstSetupComplete(
          sync_service->GetUserSettings()->IsFirstSetupComplete();
 }
 
-bool BraveSyncWorker::ResetSync(
+void BraveSyncWorker::ResetSync(
     JNIEnv* env,
     const base::android::JavaParamRef<jobject>& jcaller) {
-  syncer::SyncService* sync_service = GetSyncService();
+  auto* sync_service =
```

Comment:
nit - you could cast in `GetSyncService`

Reply:
addressed in https://github.com/brave/brave-core/pull/6565/commits/3ce15e18d2629cc8017b6a39459249389f0d7cd8